modules/rest_api/controllers/services/rest.php

```diff
@@ -924,7 +924,7 @@ private function getEsPostData($postObj, $format, $file) {
           $fields[] = 'event.date_start';
           $fields[] = 'event.date_end';
         }
-        elseif ($field === '#lat_lon#' || $field === '#lat#' || $field === '#lon#' || $field === '#dec_lat#' || $field === '#dec_lon#') {
+        elseif ($field === '#lat_lon#' || $field === '#lat#' || $field === '#lon#') {
```

Comment:
Need to use regexp to check if field matches #lat# or #lat:option# etc here, simple string match no longer good enough?